pydbtools/_wrangler.py

```diff
@@ -52,7 +56,7 @@ def wrapper(*args, **kwargs):
 
         # Create a db flag
         database_flag = "database" in sig.parameters and (
-            argmap.get("database", "__temp__").lower() == "__temp__"
+            argmap.get("database", "__temp__") in ["__temp__", "__TEMP__"]
```

Comment:
Why did this need to change?

(Was this something I asked for, have a vague memory of doing so but it might have been somewhere else)

Reply:
The commit for that is "Fix lower being applied to database=None", I think it was a bug identified running one of the notebooks.